DesiMock

```diff
@@ -0,0 +1,195 @@
+from astropy.io import fits
+import numpy as np
+
+
+class Sightline(object):
```

Comment:
Please modify the `Sightline` object in data_model/Sightline.py instead of generating a new object.  Thanks.

Reply:
Ok, I see you do that below.  Therefore, you don't need this code at all.  Please remove.